.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,13 @@
-# Read and remove this section before submitting
+# Read this section before submitting
 
 * Ensure your PR follows the [External Contribution Guidelines](https://github.com/leanprover/lean4/blob/master/CONTRIBUTING.md).
 * Please make sure the PR has excellent documentation and tests. If we label it `missing documentation` or `missing tests` then it needs fixing!
-* Add the link to your `RFC` or `bug` issue below.
+* Include the link to your `RFC` or `bug` issue in the description.
```

Comment:
Should we remove the default text below the line then? It's not really the format we want to end up in the commit message

Reply:
Is there a format that makes sense to suggest, or should we just remove it all an leave it to common sense?

Reply:
I suppose we may as well suggest "Closes #<issue>" to make sure they are linked